ext/civicrm_admin_ui/managed/SavedSearch_Contact_Summary_Memberships.mgd.php

```diff
@@ -0,0 +1,493 @@
+        'saved_search_id.name' => 'Contact_Summary_Memberships',
+        'type' => 'table',
+        'settings' => [
+          'description' => E::ts('Active Memberships'),
+          'sort' => [
+            [
+              'id',
+              'DESC',
+            ],
+          ],
+          'limit' => 0,
+          'pager' => FALSE,
+          'placeholder' => 5,
+          'columns' => [
+            [
+              'type' => 'field',
+              'key' => 'membership_type_id:label',
+              'dataType' => 'Integer',
+              'label' => E::ts('Membership'),
+              'sortable' => TRUE,
+              'rewrite' => '[membership_type_id:label] [IF_owner_membership_id_owner_membership_id]',
+            ],
+            [
+              'type' => 'field',
+              'key' => 'join_date',
+              'dataType' => 'Date',
+              'label' => E::ts('Member Since'),
+              'sortable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'start_date',
+              'dataType' => 'Date',
+              'label' => E::ts('Membership Start Date'),
+              'sortable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'end_date',
+              'dataType' => 'Date',
+              'label' => E::ts('Membership Expiration Date'),
+              'sortable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'status_id:label',
+              'dataType' => 'Integer',
+              'label' => E::ts('Status'),
+              'sortable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'source',
+              'dataType' => 'String',
+              'label' => E::ts('Membership Source'),
+              'sortable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'IF_contribution_recur_id_contribution_recur_id',
+              'dataType' => 'String',
+              'label' => E::ts('Auto-Renew'),
+              'sortable' => TRUE,
+              'title' => NULL,
+              'cssRules' => [],
+              'rewrite' => '',
+              'icons' => [
+                [
+                  'icon' => 'fa-check',
+                  'side' => 'left',
+                  'if' => [
+                    'IF_contribution_recur_id_contribution_recur_id',
+                    'IS NOT EMPTY',
+                  ],
+                ],
+              ],
+            ],
+            [
+              'type' => 'field',
+              'key' => 'owner_membership_id',
+              'dataType' => 'Integer',
+              'label' => E::ts('Related'),
+              'sortable' => TRUE,
+              'rewrite' => "{crmAPI var='owner_count' entity='Membership' action='getcount' owner_membership_id=[id]}\n{if \$owner_count}\n  {\$owner_count} {ts}created{/ts}\n{else}\n  {ts}N/A{/ts}\n{/if}",
```

Comment:
Can this be done with a join + group_by id?

Reply:
Seems we dont yet have a [self join on SK](https://github.com/civicrm/civicrm-core/blob/master/ext/search_kit/Civi/Search/Admin.php#L312-L317)?